src/xdocs/checks/javadoc/writetag.xml

```diff
@@ -201,13 +203,14 @@ public class Test { // violation as required tag is missed
 * Some class
 * @since 1.2
 */
-public class Test {
+public class Example4 {
+  // violation 2 lines below 'Type Javadoc tag @since'
   /** some doc
-  * @since violation
+  * @since
```

Comment:
Please make all javadoc to be formatted as:
```
/** some doc
   * @since
   */
```
Leading * is under first * of first line of comment.
Please apply same for class javadoc.

Reply:
Done